server/service/service.go

```diff
@@ -34,6 +33,11 @@ func New(conf *config.Config) (*Service, error) {
 		return nil, fmt.Errorf("failed to create PanaceaGRPCClient: %w", err)
 	}
 
+	_, err = panaceaClient.GetRegisteredDataValidator(validatorAccount.GetAddress())
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	registeredDataVal, err = panaceaClient.GetRegisteredDataValidator(validatorAccount.GetAddress())
	if err != nil {
		return nil, err
	} else if registeredDataVal == nil {
	        return nil, fmt.Errorf("this data validator is not registered in Panacea yet")
	}
```

Reply:
Plus, please change the title of this PR: `Check whether the data validator is registered in Panacea`.

Reply:
Ok, I'll rename it